src/pal/pal_apple.h

```diff
@@ -22,6 +24,58 @@ namespace snmalloc
      * should add any required features.
      */
     static constexpr uint64_t pal_features = PALBSD::pal_features;
+
+    // OS specific function for zeroing memory with ID 241.
```

Comment:
Please make this a proper doc comment, don't mention the magic number, do say what the magic number represents.